### ISSUE TITLE
Claiming incidents via text

### ISSUE THREAD

**Issue by jgoldrich** [linkedin/iris#583]
Is there a way to claim an incident via respond to the text message? I've tried replying with `2`, `claim`, `claim_all`, and `batch_claim` and none of those worked.

Thanks!
Jake

**Comment by diegocepedaw** [comment]
Yes you can claim via SMS, you have to respond with "$INCIDENT_ID claim" to claim the incident

**Comment by jgoldrich** [comment]
@diegocepedaw the text I get just has the message id which is different than the claim id. nevertheless, `$INCIDENT_ID claim` and `$MESSAGE_ID claim` both didn't seem to work for me

**Comment by lukepeeler** [comment]
Bump for the above -- this would be super helpful for our workflow

**Comment by jrgp** [comment]
You can also reply with `claim all` to claim all active incidents which you received messages for

**Comment by jgoldrich** [comment]
> You can also reply with `claim all` to claim all active incidents which you received messages for

this didn't work for me. any other ideas?
